sql/mvtbl.sql

```diff
@@ -20,3 +20,21 @@ $$
 		RETURN res;
 	END;
 $$ LANGUAGE plpgsql STRICT;
+
+CREATE FUNCTION mvtbl(tbl text, tblspace text, retries int, sleep_sec int DEFAULT 10)
+RETURNS bigint AS
+$$
+	DECLARE
+		res bigint;
+	BEGIN
+		FOR i in 1..retries LOOP
```

Comment:
This means that if we reach "retries" attempts, the function will return `NULL` and not an error.
Then the caller would have to make a special case for `NULL` return.
Wouldn't it make more sense to still report an error if we reach that case?

Reply:
sure makes sense, fixed in 3c301a4